src/couch/src/couch_server.erl

```diff
@@ -238,13 +238,16 @@ init([]) ->
     couch_util:set_process_priority(?MODULE, high),
 
     % Mark pluggable storage engines as a supported feature
-    config:enable_feature('pluggable-storage-engines'),
+    % Not implemented on FDB
+    % config:enable_feature('pluggable-storage-engines'),
 
     % Mark partitioned databases as a supported feature
-    config:enable_feature(partitioned),
+    % Not implemented on FDB
+    % config:enable_feature(partitioned),
 
     % Mark being able to receive documents with an _access property as a supported feature
-    config:enable_feature('access-ready'),
```

Comment:
We are _access ready technically so let's keep this one enabled. Though couch_server is probably the wrong place for it. We'd want it in fabric2_server eventually or other such place.

https://github.com/apache/couchdb/blob/prototype/fdb-layer/src/couch/src/couch_doc.erl#L304

Reply:
Updated.